tests/gnrc_sock_dns/README.md

```diff
@@ -14,7 +14,6 @@ Start dnsmasq (in another console):
 
     $ sudo dnsmasq -d -2 -z -i tap0 -q --no-resolv \
         --dhcp-range=::1,constructor:tap0,ra-only \
-        --listen-address 2001:db8::1 \
```

Comment:
I mentioned this in the commit that does this change: this bind causes (at least my version of `dnsmasq`) to send RAs from that address. However, sending RAs from a non-link-local address is invalid.